tests/contest/runtimetest/src/utils.rs

```diff
@@ -1,22 +1,48 @@
 use std::fs;
-use std::fs::{metadata, symlink_metadata};
+use std::fs::{metadata, symlink_metadata, OpenOptions};
+use std::io::Read;
 use std::os::unix::prelude::MetadataExt;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use std::process::Command;
 
 use nix::sys::stat::{stat, SFlag};
 
-fn test_file_read_access(path: &str) -> Result<(), std::io::Error> {
-    let _ = std::fs::OpenOptions::new()
-        .create(false)
-        .read(true)
-        .open(path)?;
-    Ok(())
+// It means the file or directory is readable
+type Readable = bool;
```

Comment:
Why did we newtype this? Not opposed to do so, but because we are not implementing any new methods on this, and the actual values we return are just bool values, is there any special advantage for making a newtype here instead of just using plain bool?

Reply:
Sorry for the late reply.We were discussing the following link:https://github.com/youki-dev/youki/pull/2950#discussion_r1932074342
The background of this change is that the bool value was unclear in its meaning, so we addressed it in this way.